test/unit/fixtures/sample-auth.js

```diff
@@ -0,0 +1,286 @@
+module.exports = {
+    v1: {
+        id: 'd497d10e-e280-8c83-709a-a4d4ea12ad14',
+        name: 'AuthTest',
+        description: 'Sample auth tests',
+        order: [
+            '951fc3e8-c6b6-5c19-9f69-4e7499b3127f',
+            '951fc3e8-c6b6-5c19-9f69-4e7499b3127g'
+        ],
+        folders: [],
+        folders_order: [],
+        requests: [
+            {
+                id: '951fc3e8-c6b6-5c19-9f69-4e7499b3127f',
+                headers: 'Authorization: Bearer wkjehbxoqnunc2k3',
+                url: 'http://echo.getpostman.com/auth/bearer',
+                preRequestScript: '',
+                method: 'GET',
+                data: [],
+                queryParams: [],
+                rawModeData: '',
+                headerData: [{
+                    key: 'Authorization',
+                    value: 'Bearer wkjehbxoqnunc2k3'
+                }],
+                dataMode: 'raw',
+                pathVariableData: [],
+                // eslint-disable-next-line max-len
+                tests: 'var response = JSON.parse(responseBody); tests["Bearer auth should pass"] = response.status === "pass";',
+                currentHelper: 'bearerAuth',
+                helperAttributes: {
+                    id: 'bearer',
+                    token: 'wkjehbxoqnunc2k3'
+                },
+                name: 'test bearer auth success',
+                collectionId: 'd497d10e-e280-8c83-709a-a4d4ea12ad14',
+                responses: []
+            },
+            {
+                id: '951fc3e8-c6b6-5c19-9f69-4e7499b3127g',
+                // eslint-disable-next-line max-len
+                headers: 'Authorization: NTLM TlRMTVNTUAADAAAAGAAYAFYAAAAYABgAbgAAAAAAAABIAAAADgAOAEgAAAAAAAAAVgAAAAAAAACGAAAABYKIogUBKAoAAAAPcABvAHMAdABtAGEAbgCDJfCgEQC+3wAAAAAAAAAAAAAAAAAAAAB6O2T1blvWpb/pqHtSdqcZ/A34nPBZe20=',
+                url: 'http://echo.getpostman.com/auth/ntlm',
+                preRequestScript: '',
+                method: 'GET',
+                data: [],
+                queryParams: [],
+                rawModeData: '',
+                headerData: [{
```

Comment:
It is okay to have it in the tests, but ideally, collection requests do not have auth and the header generated by the auth.

Reply:
This ensures that header transformations are not affected by auth (as was also the case with the examples themselves).